packages/arb-token-bridge-ui/tests/e2e/specs/importToken.cy.ts

```diff
@@ -7,14 +7,19 @@ import {
   importTokenThroughUI
 } from '../../support/common'
 
-const ERC20TokenAddressL1 = Cypress.env('ERC20_TOKEN_ADDRESS_L1')
-const ERC20TokenAddressL2 = Cypress.env('ERC20_TOKEN_ADDRESS_L2')
+const ERC20TokenAddressL1: string = Cypress.env('ERC20_TOKEN_ADDRESS_L1')
+const ERC20TokenAddressL2: string = Cypress.env('ERC20_TOKEN_ADDRESS_L2')
 
 describe('Import token', () => {
   // we use mainnet to test token lists
   // need low balance check to bypass a mainnet popup
   let isLowBalanceMainnet: boolean
 
+  afterEach(() => {
+    // after all assertions are executed, logout and reset the account
```

Comment:
If we're looking for `cy.logout` to be executed after all the assertions are completed (and not after every assertion), would it make more sense to have it under `after`, instead of `afterEach`?

Reply:
Okay, so checking further in the code, I think you mean after **every** assertion. Btw why is login-logout needed after every assertion?

Reply:
Sure, let's add it to a new issue in github.